examples/service_processor_apriltag_detector.json

```diff
@@ -0,0 +1,43 @@
+{
+	"service_type": "processor_apriltag_detector",
+	"instance_name": "apriltag_detector",
+	"subtopic": "pose_transformed",
+	"interval_ms": 30,
+	"processor_apriltag_detector_topic": "realm/d/myuser/mydevice/sensors/pose",
```

Comment:
Remove, not in use.

Reply:
The class initialization doesn't seem to work without `"instance_name"` and when I don't include a value for `"interval_ms"`, `fetch()` only seems to get called once.

Reply:
Also, `"subtopic"` seems to be required. I got rid of `"processor_apriltag_detector_topic"`, though.

Reply:
Shoot sorry my bad, yes just remove `processor_apriltag_detector_topic`.